utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -25,10 +24,16 @@ case class KafkaConfig(kafkaAddress: String,
                        // and all topics related to this config require both key and value schema definitions.
                        useStringForKey: Boolean = true,
                        schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig(),
-                       avroAsJsonSerialization: Option[Boolean] = None
-                      ) {
+                       avroAsJsonSerialization: Option[Boolean] = None,
+                       kafkaAddress: Option[String] = None) {
 
-  def schemaRegistryClientKafkaConfig = SchemaRegistryClientKafkaConfig(kafkaProperties.getOrElse(Map.empty), schemaRegistryCacheConfig, avroAsJsonSerialization)
+  // delete this method, make `kafkaProperties` required and use it instead, when `kafkaAddress` is removed
+  def definedKafkaProperties = kafkaAddress.map(bs => Map("bootstrap.servers" -> bs)).getOrElse(Map.empty) ++ kafkaProperties.getOrElse(Map.empty)
+
+  //is that check needed? Sometimes we do not need this property eg. to create SchemaRegistryClient
+  require(definedKafkaProperties.contains("bootstrap.servers"), "Missing 'bootstrap.servers' property in kafkaProperties")
```

Comment:
I'm not sure if it's needed - in almost all settings except for simple development/demo you have to set other properties anyway...